docs/api.yml

```diff
@@ -67,6 +67,7 @@ sidebar:
       - "api-reference/source_connectors/pubsub"
       - "api-reference/source_connectors/redshift"
       - "api-reference/source_connectors/s3"
+      - "api-reference/source_connectors/secret"
```

Comment:
Lets add this section in the beginning since it is being used for other connectors